src/els_uri.erl

```diff
@@ -52,7 +52,7 @@ uri(Path) ->
            true -> els_utils:to_binary(string:replace(Path, "\\", "/"));
            false -> Path
          end,
-  <<"file:///", Path1/binary>>.
+  <<"file://", Path1/binary>>.
```

Comment:
This is definitely wrong if you don't guarantee an absolute path.

Reply:
I may be wrong, but as far as I could see, the clients always send absolute paths to the server. Isn't this the case?

Reply:
It is, but this is the other way round (from Erlang to client). I've found the "official" logic and will try to implement it here: https://github.com/microsoft/vscode-uri/blob/master/src/index.ts#L1005-L1026